src/python/pants/core/goals/test.py

```diff
@@ -500,6 +510,35 @@ async def run_tests(
     return Test(exit_code)
 
 
+def _format_test_summary(result: TestResult, run_id: RunId, console: Console) -> str:
+    """Format the test summary printed to the console."""
+
+    SourceMap = {
```

Comment:
Not sure if this additional mapping is needed, or if the wording can be just be changed on the rust side.